src/context/common.jsonld

```diff
@@ -38,6 +38,7 @@
     "matchMedia": {"@id": "ren:matchMedia"},
     "css": {"@id": "ren:CssStyle#"},
     "backgroundImage": {"@id": "ren:backgroundImage", "@type": "@id"},
+    "gridCols" : {"@id":, "ren:gridCols, "@type": "@id},
```

Comment:
This is not needed. The schema should be defined and referenced as `http://rengular.js.org/schema/ChoiceMenu#gridCols`, which could be written in short as `ChoiceMenu:#gridCols` with the context `https://rengular.js.org/context/common.jsonld` in usage.